### PR TITLE
Implement famous player list

### LINKED ISSUE #764: /list -f
/list -f = View Fake famous player list...

Like... Notch, Dan, dinnerbone, etc.

What You think of this?

### PR DESCRIPTION
**Pull request by DUFCLiam**
#764

### CODE REVIEW

File: src/me/StevenLawson/TotalFreedomMod/Commands/Command_list.java [hunk 1]
```diff
@@ -12,12 +12,13 @@
...
 
 @CommandPermissions(level = AdminLevel.ALL, source = SourceType.BOTH)
-@CommandParameters(description = "Lists the real names of all online players.", usage = "/<command> [-a | -i]", aliases = "who")
+@CommandParameters(description = "Lists the real names of all online players.", usage = "/<command> [-a | -i | -f]", aliases = "who")
 public class Command_list extends TFM_Command
 {
     private static enum ListFilter
     {
         ALL,
+        FAKE,
```
Comment (JeromSar) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_list.java]: `FAMOUS` probably what is meant here. :P

File: src/me/StevenLawson/TotalFreedomMod/Commands/Command_list.java [hunk 2]
```diff
@@ -52,6 +53,10 @@ else if ("-i".equals(args[0]))
             {
                 listFilter = ListFilter.IMPOSTORS;
             }
+            if ("-f".equals(args[0]))
```
Comment (JeromSar) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_list.java]: Missing else

File: src/me/StevenLawson/TotalFreedomMod/Commands/Command_list.java [hunk 3]
```diff
@@ -72,6 +77,11 @@ else if ("-i".equals(args[0]))
         final List<String> names = new ArrayList<String>();
         for (Player player : server.getOnlinePlayers())
         {
+            if (listFilter == ListFilter.FAKE && !TFM_ConfigEntry.UNBANNABLE_USERNAMES(player))
+            {
+                continue;
+            }
+            
             if (listFilter == ListFilter.ADMINS && !TFM_AdminList.isSuperAdmin(player))
```
Comment (JeromSar) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_list.java]: While you're at it, prefix this line with `else` too.

**Comment by JeromSar** [comment]
Yet again, another untested PR. This one does not compile. How embarrassing.

**Comment by ReelPattytothewatty** [comment]
@JeromSar Tested it, does not work!

**Pull request closed without merging**